### ISSUE TITLE
Please update for NextCloud 15

### ISSUE THREAD

**Issue by MagicFab** [cweiske/grauphel#70]
NextCloud 15 is out, Grauphel can't be used in it as it's marked for compatibility until NC14 only.

I can help testing if any significant changes are needed.

**Comment by cweiske** [comment]
Yes, this is that way because I did not test on NC 15 yet.
You could help by trying the version on NC15 yourself:

1. Download the `.tar.gz` file from http://cweiske.de/grauphel.htm#download
2. Extract it in your `nextcloud/apps/` directory
3. Adjust `appinfo/info.xml` to have a nextcloud `max-version` of 15
4. Enable grauphel in the nextcloud apps manager
5. Test it
6. Report back, so that I can either fix the issues you found or increase the max version and re-release grauphel.

(Instructions taken from https://github.com/cweiske/grauphel/issues/54#issuecomment-361166105)

**Comment by MagicFab** [comment]
Thank you for the instructions reminder. I got some time today to test this and used a new Debian Testing VM, d/l and enabled Grauphel, updated the max-version to 15.

* http://$IP/nextcloud/index.php/apps/grauphel/ displayed instructions as expected
* I configured Tomboy in Debian Testing and TomDroid on Android, registered them as using Grauphel on the test server
* I sync'ed à 40 existing notes, then proceeded to delete, edit, add new notes. Everything worked as expected.

Using loglevel 0, nextcloud.log didn't show anything about Grauphel.
